src/app/pages/NFTInstanceDashboardPage/InstanceImageCard.tsx

```diff
@@ -99,7 +99,7 @@ export const InstanceImageCard: FC<InstanceImageCardProps> = ({ isFetched, isLoa
             minHeight: imageSize,
           }}
         >
-          {isLoading && <Skeleton variant="rectangular" width={imageSize} height={imageSize} />}
+          {isLoading && <Skeleton className={`w-[${imageSize}] h-[${imageSize}]`} />}
```

Comment:
I suspect this doesn't work, just like https://github.com/oasisprotocol/explorer/pull/2106/files#r2252699452

Reply:
it does not support dynamic values